python/source/create.rst

```diff
@@ -7,6 +7,68 @@ Tensors and all other Arrow entities.
 
 .. contents::
 
+Creating Arrays
+===============
+
+Arrow keeps data in continuous arrays optimised for memory footprint
+and SIMD analyses. In Python it's possible to build :class:`pyarrow.Array`
+starting from Python ``lists`` (or sequence types in general),
+``numpy`` arrays and ``pandas`` Series.
+
+.. testcode::
+
+    import pyarrow as pa
+
+    array = pa.array([1, 2, 3, 4, 5])
+
+.. testcode::
+
+    print(array)
+
+.. testoutput::
+
+    [
+      1,
+      2,
+      3,
+      4,
+      5
+    ]
+
+Arrays can also provide a ``mask`` to specify which values should
```

Comment:
I don't know if it's worth mentioning but the `mask` must be a numpy array (e.g. typical python list won't work)

Reply:
It's something I struggled a bit with, I wasn't sure if I should have explicitly mentioned that or not. Mostly for the reason that I would like to expand that to support ``pyarrow.Array`` too as masks ( https://issues.apache.org/jira/browse/ARROW-13883 ). So in the end I thought about not mentioning the constraint so that we don't leave behind a wrong information when the constraint is lightened

Reply:
If you're planning on expanding support then I agree it should not be mentioned here.